index.html

```diff
@@ -198,15 +198,28 @@ <h3>Easy to install</h3>
 
             <p>
               The easiest way to install CuPy is to use pip.
-              CuPy also supports various versions of CUDA and cuDNN.
-              The installer automatically detects installed versions of CUDA and cuDNN in your environment.
-              <a href="https://docs-cupy.chainer.org/en/stable/install.html">See details</a>.
+              CuPy provides Wheels (precompiled binary packages) for the recommended environments. These packages include cuDNN and NCCL.
+              <a href="https://docs-cupy.chainer.org/en/stable/install.html#install-cupy">See details</a>.
+
+              And, CuPy can be installed from source code.
+              The install script in the source code automatically detects installed versions of CUDA, cuDNN and NCCL in your environment.
+              <a href="https://docs-cupy.chainer.org/en/stable/install.html#install-cupy-from-source">See details</a>.
             </p>
          </div>
 
          <div class="six columns feature-media left">
            <!-- <img src="images/feature-image-3.png" alt="" /> -->
            <pre><code class="language-bash">
+(For CUDA 8.0)
+$ pip install cupy-cuda80
+
+(For CUDA 9.0)
+$ pip install cupy-cuda90
+
+(For CUDA 9.1)
+$ pip install cupy-cuda91
+
+(Install CuPy from source)
 % pip install cupy

```

Comment:
Only this line uses “%” mark for indicating a prompt, but I think it’s better to unify the character.